packages/web-react/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { ComponentMeta } from '@storybook/react';
+import Dropdown from './Dropdown';
+
+export default {
+  title: 'Components/Dropdown',
+  component: Dropdown,
+} as ComponentMeta<typeof Dropdown>;
+
```

Comment:
Looks like there is some CSS missing in Storybook examples… 🤔 

![obrazek](https://user-images.githubusercontent.com/5614085/199208539-6c32ffd7-10a2-4983-9d29-0efecbe29a0f.png)

Reply:
Fixed